agent/host_agent_test.go

```diff
@@ -567,7 +568,7 @@ var _ = Describe("Agent", func() {
 			cleanup(runner.Context, byoHostContainer, ns, agentLogFile)
 		})
 
-		It("should enable the SecureAccess feature gate", func() {
+		It("should enable the bootstrap flow", func() {
```

Comment:
```suggestion
		It("should enable the bootstrap kubeconfig flow", func() {
```
Just to maintain consistency all along.

Reply:
thanks, fixed.